### PR TITLE
Elb v3 pool

### PR DESCRIPTION
**Pull request by anton-sidelnikov**

### CODE REVIEW

File: octavia_proxy/api/drivers/elbv3/driver.py [hunk 1]
```diff
@@ -124,6 +125,12 @@ def listeners(self, session, project_id, query_filter=None):
         if not query_filter:
             query_filter = {}
 
+        if 'id' in query_filter:
+            query_filter['name'] = self.listener_get(
+                project_id=project_id, session=session,
+                lsnr_id=query_filter['id']).name
+            query_filter.pop('id')
+
```
Comment (outcatcher) [inline octavia_proxy/api/drivers/elbv3/driver.py:133]: Shouldn't here function return just `self.listener_get` result?

File: octavia_proxy/api/drivers/elbv3/driver.py [hunk 2]
```diff
@@ -194,6 +201,64 @@ def listener_delete(self, session, listener):
...
+        LOG.debug('Fetching pools')
+
+        if not query_filter:
+            query_filter = {}
+
+        if 'id' in query_filter:
+            query_filter['name'] = self.pool_get(
+                project_id=project_id, session=session,
+                pool_id=query_filter['id']).name
+            query_filter.pop('id')
```
Comment (outcatcher) [inline octavia_proxy/api/drivers/elbv3/driver.py:214]: Same as the previous comment

File: octavia_proxy/api/v2/controllers/pool.py [hunk 3]
```diff
@@ -79,4 +223,4 @@ def _lookup(self, pool_id, *remainder):
         Verifies that the pool passed in the url exists, and if so decides
         which controller, if any, should control be passed.
         """
-        pecan_abort(501)
+        return None
```
Comment (outcatcher) [inline octavia_proxy/api/v2/controllers/pool.py:226]: Is this a valid return?

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: octavia_proxy/api/v2/controllers/pool.py (hunk 3 above)
Comment (anton-sidelnikov) [reply octavia_proxy/api/v2/controllers/pool.py:226]: yep

**Review by Polina-Gubina — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Review by Polina-Gubina — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**